core/ev-admin.php

```diff
@@ -423,7 +425,7 @@ function fetch_new_videos( $update_hosts, $update_author ) {
 
       if( $update_author == null ){
         // fetch all hosts, all authors
-        //error_log( 'update author was null. <br />' );
+        error_log( 'update author was null. <br />' );
```

Comment:
This isn't strictly an error, just an indication that there are no new videos. We shouldn't put an error in the logs.